trackio/__init__.py

```diff
@@ -188,4 +131,104 @@ def show(project: str | None = None):
     dashboard_url = base_url + f"?project={project}" if project else base_url
     print(f"* Trackio UI launched at: {dashboard_url}")
     webbrowser.open(dashboard_url)
-    block_except_in_notebook()
+    utils.block_except_in_notebook()
+
+
+def import_csv(
```

Comment:
I wonder if we should put this in its own file `import.py` -- especially if we eventually do #58 as well.